docs/features/software-templates/builtin-actions.md

```diff
@@ -14,3 +14,39 @@ Azure, GitLab and Bitbucket.
 A list of all registered actions can be found under `/create/actions`. For local
 development you should be able to reach them at
 `http://localhost:3000/create/actions`.
+
+### Migrating from `fetch:cookiecutter` to `fetch:template`
```

Comment:
Open to suggestions for other places this could live!

Reply:
Doesn't look like there's an established pattern for migration docs, but I like this idea of colocating it with related docs and linking to it from the changeset 👍